tests/unit/test_match_users.py

```diff
@@ -105,12 +150,24 @@ def test_single_match(self):
 
         # Check for correct response
         res_json = response.get_json()
-        assert sorted(res_json[0].items()) == sorted(self.journey_one.items())
+
+        for maps in res_json:
+            currUserID = maps["id"]
+            for key, value in maps.items():
```

Comment:
Incorrect equivalence checks. Only checks for the correct email address.

Removes journey correctness assertion in response.

I hardcoded the API response to journey five and still, all the tests are passing. Lol

Reply:
Your understanding is slightly incorrect. The equivalence checks are correct as long as the response returned is not an empty list. Can fix for the single match case by adding an assertion for the length of res_json `assert Len(res_json)==1`. 

The maps however do check everything, not just the email.

Reply:
In this case by the maps I'm referring to the returned json of the user in the format the db would return it. This method is mocked however since we aren't establishing a connection to the db, I believe this is the correct way to inject dependencies.